Common/Extensions.cs

```diff
@@ -230,6 +230,15 @@ public static long ToInt64(this string str)
         }
 
         /// <summary>
+        /// Check if a number is NaN or equal to zero
+        /// </summary>
+        /// <param name="value">The double value to check</param>
+        public static bool IsNaNOrZero(this double value)
+        {
+            return double.IsNaN(value) || Math.Abs(value) < double.Epsilon;
```

Comment:
Is this check intended to include `double.Epsilon`? It looks like this is really doing:
```C#
return double.IsNan(value) || value == 0;
```
Since no positive double can exist between `double.Epsilon` and 0.

No change needed, the comments express this behavior clearly, just thinking out loud.